docs/developer-docs/web-apps/verifiable-credentials/issuer.mdx

```diff
@@ -0,0 +1,182 @@
+---
+keywords: [intermediate, tutorial, identity, credentials, verifiable credentials, issuer]
+---
+
+import { MarkdownChipRow } from "/src/components/Chip/MarkdownChipRow";
+import useBaseUrl from "@docusaurus/useBaseUrl";
+
+# Become an issuer
+
+<MarkdownChipRow labels={["Intermediate", "Tutorial"]} />
+
+## Overview
+
+An **issuer** is a service or app that can issue a verifiable credential to a user. To become an issuer, your canister must implement the verifiable credential interface using the issuer API. The issuer API is used to issue verifiable credentials, while another API, the identity provider API, is used to communicate with an identity provider such as Internet Identity. End-users communicate with the identity provider for authentication and to consent to being issued a verifiable credential. An identity provider is also used to relay information between the issuer and the [relying party](relying-party.mdx) when a verifiable credential is requested by the relying party.
+
+Internet Identity is used as the identity provider in this tutorial.
+
+## Using the issuer API
+
+To use the issuer API, your canister must support four steps that are required to issue a verifiable credential:
+
+1. Obtain a consent message (API endpoint `vc_consent_message`): Receive a message of consent from identity provider to issue a user a verifiable credential.
+
+2. Implement a derivation origin (API endpoint `derivation_origin`): Configure an alternative origin URL that the identity provider may send the request from and still be considered valid.
+
+3. Prepare the credentials (API endpoint `prepare_credential`): Prepare the credentials for the requested user.
+
+4. Issue the credential (API endpoint `get_credential`): Verify the credential and officially issue it to the user.
+
+### Step 1: Obtain a consent message (`vc_consent_message`)
```

Comment:
I didn't review this and the following sections because I believe there was a misunderstanding on the issuer's role.